WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -296,6 +296,8 @@ class PostListViewController: AbstractPostListViewController, UIViewControllerRe
         updateGhostableTableViewOptions()
         ghostableTableView.reloadSections([0], with: .automatic)
 
+        postsViewButtonItem.accessibilityLabel = NSLocalizedString("List style", comment: "List display style")
```

Comment:
Usually, the `comment` would also describe where it can be found and how it is used. My suggestion would be something along the lines of:

```
The accessibility label for the list style button in the Post List.
```

It gives some context for the translator and also helps them find that button in the app. 

Feel free to improve the other `NSLocalizedString` you created in this PR too. 🙂

Reply:
Alright, I've re-written the 4 `NSLocalizedString`s. What do you think of them?

Reply:
They look great! 👍 Thank you!